components/prize_pool/commons/prize_pool.lua

```diff
@@ -537,6 +544,59 @@ function PrizePool:setLpdbInjector(lpdbInjector)
 	return self
 end
 
+function PrizePool:_storeLpdb()
+	local lpdbTournamentData = {
+		tournament = Variables.varDefault('tournament_name'),
+		parent = Variables.varDefault('tournament_parent'),
+		series = Variables.varDefault('tournament_series'),
+		shortname = Variables.varDefault('tournament_tickername'),
+		startdate = Variables.varDefaultMulti('tournament_startdate', 'tournament_sdate', 'sdate', ''),
+		mode = Variables.varDefault('tournament_mode'),
+		type = Variables.varDefault('tournament_type'),
+		liquipediatier = Variables.varDefault('tournament_liquipediatier'),
+		liquipediatiertype = Variables.varDefault('tournament_liquipediatiertype'),
+		-- publishertier = ,
+		icon = Variables.varDefault('tournament_icon'),
+		icondark = Variables.varDefault('tournament_icondark'),
+		game = Variables.varDefault('tournament_game'),
+	}
+
+	local setWeight = function(lpdbEntry)
+		lpdbEntry.weight = Template.safeExpand(
```

Comment:
maybe something like (pseudo code)
```suggestion
		local weightArgs = {money, tier, place, type, tiertype, opponenttype}
		lpdbEntry.weight = self.weightFunction and self.weightFunction(weightArgs) or Template.safeExpand(frame, weightArgs)
```

Reply:
Well, you can just use the LPDB Injector (once it exists) to set a different weight. Might just move all weights to custom (via LPDB Injector)